### DISCUSSION TITLE
Lab6 Firmware 問題

### DISCUSSION THREAD

**Discussion by Wujuichen** [bol-edu/HLS-SOC-Discussions#156] [Q&A]
看完lab6的影片講解後，有以下疑問:不太清楚4份C code合起來的意義? 還有ISR不太清楚是要寫在4份C code的哪個部分?  感謝!!

**Comment by LingChiYang** [comment]
同學好

1. 合起來是為了final project測overall latency用，讓你知道不只要針對一個應用加速
2. ISR在這個lab只用在UART傳輸，請參考 https://github.com/bol-edu/caravel-soc_fpga-lab/blob/main/lab-wlos_baseline/firmware/isr.c

**Comment by Wujuichen** [comment]
謝謝回覆!
